lib/frr_zmq.c

```diff
@@ -263,14 +262,13 @@ static int frrzmq_write_msg(struct thread *t)
 
 	thread_add_write(t->master, frrzmq_write_msg, cbp,
 			 cb->fd, &cb->write.thread);
-	return 0;
+	return;
```

Comment:
missed, cleanup.

Reply:
same with this one. I do not want to cause a flog_err to be generated